api/receiver/create_run_test.go

```diff
@@ -44,12 +45,14 @@ func TestHandleResultsCreate(t *testing.T) {
 	req.SetBasicAuth("_processor", "secret-token")
 	w := httptest.NewRecorder()
 	mockAE := NewMockAppEngineAPI(mockCtrl)
+	mockS := checks.NewMockSuitesAPI(mockCtrl)
 	gomock.InOrder(
 		mockAE.EXPECT().authenticateUploader("_processor", "secret-token").Return(true),
 		mockAE.EXPECT().addTestRun(gomock.Any()).Return(testDatastoreKey, nil),
+		mockS.EXPECT().CompleteCheckRun(gomock.Any(), gomock.Any()).Return(true, nil),
```

Comment:
Let's be a little more rigorous here. We know the values of the two arguments and let's just expect them explicitly instead of using `gomock.Any`.

Reply:
Done.